librz/core/cmd_descs.yaml

```diff
@@ -590,9 +590,32 @@
         - name: project.rzdb
           type: RZ_CMD_ARG_TYPE_FILE
 - name: q
-  cname: cmd_quit
-  summary: Quit program with a return value
-  type: RZ_CMD_DESC_TYPE_OLDINPUT
+  summary: Quit rizin
+  subcommands:
+    - name: "q"
+      cname : cmd_quit
+      summary: quit rizin
+      args: []
+    - name: "q!"
+      cname: cmd_force_quit
+      summary: Force quit rizin
+      args: []
+    - name: "q!!"
+      cname: cmd_force_quit_without_history
+      summary: force quit rizin without saving history
+      args: []
+    - name: q
```

Comment:
Maybe instead of this we can change a bit the way this thing is done... 
What if we just keep one `q` command that optionally accept arguments? First argument is "kill process", second argument is "save project". So if you want to kill and save project you would do `q y y`. If we go this route, just remove this and add the optional arguments in https://github.com/rizinorg/rizin/pull/525/files#diff-4f94ed53f57e9c49ad8a154a2d4e3defb758c0deaf69909274e560bb29a63938R598

@XVilka WDYT?

Reply:
While it makes sense this is not how Vim commands work (Rizin commands are largely inspired by Vim).

Reply:
So you would stay with `qyy` `qyn` `qny` `qnn`?

Reply:
I think `qyy` is fine probably. Not 100% sure.

Reply:
Ok, in this case @officialcjunior you should change this to be similar to https://github.com/rizinorg/rizin/blob/dev/librz/core/cmd_descs.yaml#L884